README.md

```diff
@@ -34,3 +34,16 @@ Send a new PR to this repository on GitHub, append your application table. Radxa
 | Project Software: | NixOS |
 | Other Notes:      | - |
 
+| Project Name:     | Lemuria  |
+| ----------------- | -------- |
+| CM3 Model:        | WiFi, eMMC,  RAM >= 4GB    |
+| Project Hardware: | Currently the RPi CM4     |
+| Project Software: | Raspbian/Linux     |
+| Other Notes:      | Linumiz, would like to port Yocto on CM3 + Lemuria.   |
+|		    | Lemuria is a low cost IIOT Gateway to address various industrial needs with rich set of interfaces |
+|	            | Its technically feasible to port yocto to rockchip/rk3566, if schematics for CM3 are made available. |
+|		    | Project will be open sourced in post development and early piloting phase, currently it's not public. |
+|		    | Details and photos can be found here link-to-Linumiz-Lemuria-webpage |
```

Comment:
IMO it makes sense to post the picture and block diagram in LinkedIn, Twitter and refer the URL here.